tox.ini

```diff
@@ -93,11 +93,16 @@ commands =
     pip install -e acme[dev] -e .[dev] -e certbot-nginx
     python certbot-compatibility-test/nginx/roundtrip.py certbot-compatibility-test/nginx/nginx-roundtrip-testdata
 
+# This is a duplication of the command line in testenv:le_auto to
+# allow users to run the modification check by running `tox`
+[testenv:modification]
+commands =
+    {toxinidir}/tests/modification-check.sh
+
 [testenv:le_auto]
 # At the moment, this tests under Python 2.7 only, as only that version is
 # readily available on the Trusty Docker image.
 commands =
-    {toxinidir}/tests/modification-check.sh
```

Comment:
Was this deleted by mistake or did we have a misunderstanding? I don't think we should delete this line.